providers/terraform-cloud-scanner/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ctrlplane/terraform-cloud-scanner",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./src/index.ts"
+    }
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsx watch --clear-screen=false src/index.ts",
+    "clean": "rm -rf .turbo node_modules",
+    "format": "prettier --check . --ignore-path ../../.gitignore",
+    "lint": "eslint",
+    "typecheck": "tsc --noEmit --emitDeclarationOnly false",
+    "test": "vitest --run",
+    "test:watch": "vitest --watch",
+    "test:coverage": "vitest run --coverage"
+  },
+  "dependencies": {
+    "@ctrlplane/logger": "workspace:*",
+    "@ctrlplane/node-sdk": "workspace:*",
+    "@t3-oss/env-core": "^0.10.1",
+    "cron": "^3.1.7",
+    "dotenv": "^16.4.5",
+    "handlebars": "^4.7.8",
+    "lodash": "^4.17.21",
+    "p-retry": "^6.2.0",
+    "zod": "catalog:"
+  },
+  "devDependencies": {
+    "@ctrlplane/eslint-config": "workspace:*",
+    "@ctrlplane/prettier-config": "workspace:*",
+    "@ctrlplane/tsconfig": "workspace:*",
+    "@types/lodash": "^4.17.5",
+    "@types/node": "catalog:node22",
+    "@vitest/coverage-v8": "^2.1.1",
```

Comment:
whats this?

Reply:
For the coverage reports. 

If you run `pnpm test:coverage` without it you get this:
<img width="1031" alt="Screen Shot 2024-09-16 at 10 18 30 PM" src="https://github.com/user-attachments/assets/92ca5af7-216b-4bc8-8d7b-cb2a572215ce">

With it you get this:
<img width="780" alt="Screen Shot 2024-09-16 at 10 19 33 PM" src="https://github.com/user-attachments/assets/53ac3778-2ab3-4299-8d8a-ee23ca169ff7">